code/js/controllers/InternetRadioController.js

```diff
@@ -0,0 +1,55 @@
+(function() {
+  "use strict";
+
+  var BaseController = require("BaseController");
+
+  new BaseController({
+    siteName: "Internet Radio",
+
+    play() { this.launchPlayback(0); },
+    playNext() { this.launchPlayback(1); },
+    playPrev() { this.launchPlayback(-1); },
+
+    pause() {
```

Comment:
I'm missing something. Why override the functions when you can use the standard selector bindings?